src/app/GlobalStats/ClusterDistribution.tsx

```diff
@@ -0,0 +1,283 @@
+import React, { useState, useEffect } from 'react';
+import {
+    Card,
+    CardBody,
+    CardTitle,
+    Level,
+    LevelItem,
+    Select,
+    SelectOption,
+    SelectVariant,
+    Spinner,
+    Toolbar,
+    ToolbarContent,
+    ToolbarItem,
+    ToolbarItemVariant,
+    Pagination
+} from '@patternfly/react-core';
+import { 
+    TableComposable, 
+    Thead, 
+    Tr, 
+    Th, 
+    Tbody, 
+    Td,
+    TableVariant
+} from '@patternfly/react-table';
+import { Chart, ChartBar, ChartGroup, ChartVoronoiContainer } from '@patternfly/react-charts';
+import { TableErrorState } from '@app/Common/TableErrorState';
+import { useTranslation } from 'react-i18next';
+import { useClusterDistribution } from '@app/services/dataDistributionHook';
+import { ClusterDistributionStatsOption } from "@services/infinispanRefData";
+import { PopoverHelp } from '@app/Common/PopoverHelp';
+
+const ClusterDistribution = () => {
```

Comment:
@dpanshug please, component names and types must be named differently. Call this ClusterDistributionChart instead of ClusterDistribution which is the type name.